### PR TITLE
Feature/checkbox

### PR DESCRIPTION
**Pull request by selvagsz**
image

screen shot 2018-08-03 at 11 30 40 am

### CODE REVIEW

File: app/styles/@tradegecko/warehousing/components/_checkbox.scss [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      color: $c-disabled-check;
+    }
+
+    &__label {
+      cursor: not-allowed;
+    }
+  }
+}
+
+// .tg-check--bulk-action {
```
Comment (billybonks) [inline app/styles/@tradegecko/warehousing/components/_checkbox.scss]: any reason for not deleting this?

File: app/styles/@tradegecko/warehousing/variables/_tg-icons.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
+$tg-icon-font-family: 'TradeGecko';
+
+$tg-var-icon: \e600;
```
Comment (billybonks) [inline app/styles/@tradegecko/warehousing/variables/_tg-icons.scss]: should these icons variables be set in the icons package

File: addon/components/checkbox.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import Component from '@ember/component';
+import { PropTypes } from 'ember-prop-types';
+import prefix from '../utils/computed/prefix';
+import layout from '../templates/components/checkbox';
+
+const Checkbox = Component.extend({
```
Comment (billybonks) [inline addon/components/checkbox.js]: since this is usable for core can we use dasherized name

Comment (selvagsz) [reply addon/components/checkbox.js]: To stay in alignment with the addon ecosystem, we can tweak core to rely on `app/` namespace

File: app/styles/@tradegecko/warehousing/variables/_tg-icons.scss (hunk 2 above)
Comment (selvagsz) [reply app/styles/@tradegecko/warehousing/variables/_tg-icons.scss]: yeah I want to add scss support in tradegecko-fonts & move them there. I'll do it future prs

File: app/styles/@tradegecko/warehousing/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@mixin input-focus($glow: true) {
```
Comment (abdullacontractor) [inline app/styles/@tradegecko/warehousing/_mixins.scss]: There is a file `addon/styles/_mixins.scss` should the contents of that go here?

Comment (selvagsz) [reply app/styles/@tradegecko/warehousing/_mixins.scss]: yes! all the addons styles should move into the `/app`. Will do it in a separate pr

File: app/styles/@tradegecko/warehousing/components/_checkbox.scss [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+  }
+
+  &__label {
+    display: flex;
+    line-height: 18px;
+    cursor: pointer;
+  }
+
+  &__content-container {
+    padding-left: 5px;
```
Comment (abdullacontractor) [inline app/styles/@tradegecko/warehousing/components/_checkbox.scss]: Should this be a constant at the top of this file?

File: addon/templates/components/checkbox.hbs [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{{input
```
Comment (rchrdschfr) [inline addon/templates/components/checkbox.hbs]: Thoughts on using one-way binding instead? Something like:

```hbs
<input
  type="checkbox"
  class="tradegecko-ui-checkbox__input"
  id={{uniqueID}}
  checked={{checked}}
  disabled={{disabled}}
  name={{aria-label}}
  oninput={{action (mut checked) value="target.checked"}}
/>
```

Comment (selvagsz) [reply addon/templates/components/checkbox.hbs]: great suggest ❤️  Done 🤘

**Pull request merged**